ocaml/libs/timeslice/timeslice.ml

```diff
@@ -0,0 +1,48 @@
+(* avoid allocating an extra option every time *)
```

Comment:
Please add the copyright headers.

Reply:
This applies to all the files, not just this.